docs/external_secrets.md

```diff
@@ -0,0 +1,155 @@
+
+#### Independent Subchart Deployment
+In scenarios where subcharts are deployed independently, outside the scope of the umbrella chart, it is crucial to set the .Values.global.externalSecrets.deploy directive within the values.yaml file for each specific service. Additionally, to facilitate the creation of a Secret Store capable of authenticating with AWS Secrets Manager, the .Values.global.externalSecrets.separateSecretStore should be set to true in the relevant charts. This configuration is essential for establishing proper authentication mechanisms for secret retrieval.
+
+#### Configuring Separate Secret Stores
+The .Values.global.externalSecrets.separateSecretStore setting can also be applied within the context of the umbrella chart deployment. Utilizing this setting allows for the creation of distinct Secret Stores dedicated to individual services. This approach is particularly beneficial for environments where it is preferable to limit access to Secrets Manager, ensuring that services only have access to the secrets explicitly required for their operation.
+
+## Helm IAM User
+If you are using a separate IAM user for AWS Secrets Manager please follow the below instructions: 
+
+This script Bash script at the beginning of this document should have created a secret titled "NameofIAMuser-user-secret" in your cluster. You will need to retrieve these values to input into your Helm chart for the Secret Store to authenticate with AWS Secrets Manager.
+
+
+Access Key:
+```
+kubectl get secret "your secret name" -o jsonpath="{.data.access-key}" | base64 --decode
+```
+
+
+Secret Access Key
+```
+kubectl get secret "your secret name" -o jsonpath="{.data.secret-access-key}" | base64 --decode
+```
+
+You can paste the IAM access key and secret access key in the `.Values.secrets.awsAccessKeyId`/`.Values.secrets.awsSecretAccessKey` fields in the values.yaml file for the chart(s) you would like to use external secrets for. 
+
+If you are deploying external secrets with the Gen3 umbrella chart, you can utilize a local secret to avoid pasting credentials in the values.yaml file. Just set `.global.aws.useLocalSecret.enabled` to true and supply your secret name.
+
+Please note that only some Helm charts are compatible with External Secrets currently. We hope to expand this functionality in the future. If a chart is able to use External Secrets, you can see a `.Values.externalSecrets` section in the values.yaml file.
+
+## How External Secrets Works. 
+External Secrets relies on three main resources to function properly. (The below have links to examples of each resource)
```

Comment:
This mentions links but I don't see them

Reply:
Just added those links! Thank you for mentioning this!